document/src/main/java/com/ritense/document/domain/search/AdvancedSearchRequest.java

```diff
@@ -26,8 +26,8 @@ public class AdvancedSearchRequest {
     private SearchOperator searchOperator = SearchOperator.AND;
     private AssigneeFilter assigneeFilter = AssigneeFilter.ALL;
     private List<OtherFilter> otherFilters = List.of();
-
     private Set<String> statusFilter = new HashSet<>();
+    private Set<String> caseTagsFilter = new HashSet<>();
```

Comment:
It's probably not worth it right now, but we could consider wrapping this in an object. If in the future we want to make it possible to search case tags based on AND and OR we could add that more easily. Could also be done by having another argument on root level however.